container/crio/client.go

```diff
@@ -70,16 +77,21 @@ func configureUnixTransport(tr *http.Transport, proto, addr string) error {
 
 // Client returns a new configured CRI-O client
 func Client() (crioClient, error) {
-	tr := new(http.Transport)
-	if err := configureUnixTransport(tr, "unix", CrioSocket); err != nil {
-		return nil, err
-	}
-	c := &http.Client{
-		Transport: tr,
-	}
-	return &crioClientImpl{
-		client: c,
-	}, nil
+	crioClientOnce.Do(func() {
+		tr := new(http.Transport)
+		if err := configureUnixTransport(tr, "unix", CrioSocket); err != nil {
```

Comment:
I think this can just be:
```golang
if clientErr = configureUnixTransport(...; err != nil {
    return
}
```

Reply:
@dashpole 
Your comment has been addressed.